meteor/sofie-server/data-access/repositories/mongo/mongo-rundown-repository.ts

```diff
@@ -69,4 +71,16 @@ export class MongoRundownRepository extends BaseMongoRepository implements Rundo
 			)
 		}
 	}
+	/* Todo: Decide if a similar assertions should be performed for other entities.
+	 *		 If other entities should also assert similarly, then decide if this should be moved to 'BaseMongoRepository'.
+	 *		 If moved, should also be renamed to e.g 'assertEntityExist(...)'
+	 */
+	private async assertRundownExist(rundownId: string): Promise<void> {
+		const mongoRundown: MongoRundown = (await this.getCollection().findOne({
+			_id: rundownId,
+		})) as unknown as MongoRundown
```

Comment:
No need to fetch the entire Rundown here just to discard the result immediatly. Just fetch the id and check if it's returned or not